Texassemble/texassemble.cpp

```diff
@@ -1007,10 +1025,16 @@ int __cdecl wmain(_In_ int argc, _In_z_count_(argc) wchar_t* argv[])
     case CMD_MERGE:
     case CMD_GIF:
     case CMD_ARRAY_STRIP:
+    case CMD_CUBE_FROM_HC:
+    case CMD_CUBE_FROM_VC:
+    case CMD_CUBE_FROM_VC_FNZ:
+    case CMD_CUBE_FROM_HT:
+    case CMD_CUBE_FROM_HS:
+    case CMD_CUBE_FROM_VS:
         break;
 
     default:
-        wprintf(L"Must use one of: cube, volume, array, cubearray,\n   h-cross, v-cross, v-cross-fnz, h-tee, h-strip, v-strip,\n   array-strip, merge, gif\n\n");
+        wprintf(L"Must use one of: cube, volume, array, cubearray,\n   h-cross, v-cross, v-cross-fnz, h-tee, h-strip, v-strip,\n   array-strip, merge, gif,\n   cube-from-hc, cube-from-vc, cube-from-vc-fnz,\n   cube-from-ht, cube-from-hs, cube-from-vs\n\n");
```

Comment:
This is getting unwieldy to maintain. Replace it with:

```
        wprintf(L"Must use one of:");
        for (size_t j = 0; j < std::size(g_pCommands); ++j)
        {
            if (!g_pCommands[j].name)
            {
                wprintf(L"\n");
                break;
            }

            wprintf(L" %ls", g_pCommands[j].name);
            if ((j % 4) == 3)
                wprintf(L"\n\t");
        }
        return 1;
```

Reply:
I noticed there is `PrintList`.

```
wprintf(L"Must use one of: ");
PrintList(17, g_pCommands);
return 1;
```
would be better?

Reply:
Right. ``PrintList(4, g_pCommands);`` should work (doesn't need quite as much spacing at the front.